service/key/key_test.go

```diff
@@ -10,21 +10,33 @@ import (
 // Test_Key_CAPath tests the CAPath function.
 func Test_Key_CAPath(t *testing.T) {
 	tests := []struct {
-		groupName    string
+		certificateDirectory string
+		groupName            string
+
 		expectedPath string
 	}{
 		{
-			groupName:    "xa5ly",
-			expectedPath: "/certs/xa5ly/ca.pem",
+			certificateDirectory: "/certs",
+			groupName:            "xa5ly",
+
+			expectedPath: "/certs/xa5ly-ca.pem",
```

Comment:
I would actually prefer the former structure. Why do you want to change it? `/certs/xa5ly/ca.pem` makes more sense to me. It is nice and structured and clean.

Reply:
I think the flat structure is easier to manage. We don't need to worry about directories at all.